src/utils/cli.rs

```diff
@@ -5,6 +5,7 @@
 // WARNING: This file should be only updated in a non-breaking way. CLI flags should not be
 // removed, new flags should be tested.
 // See https://github.com/parallaxsecond/parsec/issues/392 for details.
+#![allow(unused_qualifications)]
```

Comment:
Why allow here?

Reply:
I'm getting this error here. From what I understand, this is getting mixed up with unused_qualifications in the definition of structopt, so I disabled it as it's a very small file. Please let me know if this is incorrect and I should change it.

Reply:
https://github.com/parallaxsecond/parsec/actions/runs/8283203581/job/22665825642#step:5:1281